src/libstore/globals.hh

```diff
@@ -826,8 +828,17 @@ public:
           > `.netrc`.
         )"};
 
-    /* Path to the SSL CA file used */
-    Path caFile;
+    Setting<Path> caFile{
+        this, getDefaultSSLCertFile(), "ssl-cert-file",
+        R"(
+          The path of a file containing CA certificates used to
+          authenticate `https://` downloads. It defaults to the first
+          of `/etc/ssl/certs/ca-certificates.crt` and
+          `/nix/var/nix/profiles/default/etc/ssl/certs/ca-bundle.crt`
+          that exists. It can be overriden using the
```

Comment:
Such information should not be inlined, but rather the algorithmic structure represented visually where possible. Documentation is just a different representation of the code's interfaces (and sometimes offers glimpses into implementation), and should abide to the same standards for readability.

```suggestion
          authenticate `https://` downloads. The default is selected by trying in order:
          1. `/etc/ssl/certs/ca-certificates.crt`
          2. `/nix/var/nix/profiles/default/etc/ssl/certs/ca-bundle.crt`
          
          It can be overriden using the
```

What happens if none is found?